lib/manageiq/messaging/stomp/client.rb

```diff
@@ -0,0 +1,50 @@
+module ManageIQ
+  module Messaging
+    module Stomp
+      class Client < ManageIQ::Messaging::Client
+        require 'stomp'
+        require 'manageiq/messaging/stomp/common'
+        require 'manageiq/messaging/stomp/queue'
+        require 'manageiq/messaging/stomp/background_job'
+        require 'manageiq/messaging/stomp/topic'
+
+        include Common
+        include Queue
+        include BackgroundJob
+        include Topic
+
+        private *delegate(:subscribe, :unsubscribe, :publish, :to => :stomp_client)
```

Comment:
I believe if you move the `private` from line 20 up above this, then you don't need to mark it directly as private here, and thus avoid the rubocop errors.